microshift_running_apps/microshift-gitops.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-gitops"]
+= Automating application management with the GitOps controller
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-gitops
+
+toc::[]
+
+GitOps with Argo CD for {microshift-short} is a lightweight, optional add-on controller derived from the Red Hat OpenShift GitOps Operator. GitOps for {microshift-short} uses the command-line interface (CLI) of Argo CD to interact with the GitOps controller that acts as the declarative GitOps engine. You can can consistently configure and deploy Kubernetes-based infrastructure and applications across clusters and development lifecycles.
+
+[id="microshift-gitops-can-do_{context}"]
+== What you can do with the GitOps agent
+By using the GitOps with Argo CD agent with {microshift-short}, you can leverage the following principles and tasks:
+
+* Create and manage your clusters and application configuration files using the core principles of developing and maintaining software in a Git repository. You can update the single repository and GitOps automates the deployment of new applications or updates to existing ones.
+** For example, if you have 1,000 edge devices, each using {microshift-short} and a local GitOps agent, you can easily add or update an application on all 1,000 devices with just one change in your central Git repository.
+* The Git repository contains a declarative description of the infrastructure you need in your specified environment and contains an automated process to make your environment match the described state.
+** For example,
+//TODO: what can we do? set memory limits or template over networking settings, etc?
+* The repository also contains the entire state of the system so that the trail of changes to the system state are visible and auditable.
+//TODO: what's the most important value prop of this visibility for microshift customers?
```

Comment:
see above: Application Lifecycle Management - dynamically manage Application / Config of solutions with a gitops approach.